allennlp/data/vocabulary.py

```diff
@@ -38,17 +38,17 @@ class _NamespaceDependentDefaultDict(defaultdict):
     specify filters on the namespace (the key used in the ``defaultdict``), and use different
     default values depending on whether the namespace passes the filter.
 
-    To do filtering, we take a sequence of ``non_padded_namespaces``.  This is a list or tuple of
-    strings that are either matched exactly against the keys, or treated as suffixes, if the
+    To do filtering, we take a set of ``non_padded_namespaces``.  This is a set of strings
+    that are either matched exactly against the keys, or treated as suffixes, if the
     string starts with ``*``.  In other words, if ``*tags`` is in ``non_padded_namespaces`` then
     ``passage_tags``, ``question_tags``, etc. (anything that ends with ``tags``) will have the
     ``non_padded`` default value.
 
     Parameters
     ----------
-    non_padded_namespaces : ``Sequence[str]``
-        A list or tuple of strings describing which namespaces are not padded.  If a namespace
-        (key) is missing from this dictionary, we will use :func:`namespace_match` to see whether
+    non_padded_namespaces : ``Iterable[str]``
+        A set of strings describing which namespaces are not padded.  If a namespace (key)
```

Comment:
I am yet to change docstrings: "set of strings" to "set / list  / tuple of strings"